src/java/org/apache/cassandra/serializers/TimestampSerializer.java

```diff
@@ -188,7 +180,7 @@ public void validate(ByteBuffer bytes) throws MarshalException
 
     public String toString(Date value)
     {
-        return value == null ? "" : FORMATTER.get().format(value);
+        return toStringUTC(value);
```

Comment:
I like the extra resolution in the output, but was wondering why the change was necessary. Does anything rely on this being a particular format?

Reply:
I tried finding a user and the only places I saw were debugging.  When it came to CQL they used different call sites so didn't see this change reflected to users.

Could be a transitive call, so am worried about this change.  I see we support reading a ton of different patterns; mostly concerned if this is exposed in a place I have not seen yet.

Reply:
Fair enough, good to simplify if we can.